apps/web/src/components/PositionCard/index.tsx

```diff
@@ -371,7 +371,7 @@ function FullPositionCard({
             <Text color="textSubtle">{t('Share of Pool')}</Text>
             <Text>
               {poolTokenPercentage
-                ? `${poolTokenPercentage.toFixed(2) === '0.00' ? '<0.01' : poolTokenPercentage.toFixed(2)}%`
+                ? `${poolTokenPercentage.toFixed(2) === '0.00' ? '0.00' : poolTokenPercentage.toFixed(2)}%`
```

Comment:
Now you don't need to check this because you not change the value when is equal to `0.00`

```suggestion
                ? `${poolTokenPercentage.toFixed(2)}%`
```

Reply:
Good, thanks